RetailSuiteAccounting/Integration/06_ManageFixedBookings.md

```diff
@@ -0,0 +1,182 @@
+
+
+## Delete a fixed booking batch
+
+You can delete an existing fixed booking batch if it is no longer applicable.
+
+#### Prerequisites
+
+- The accounting is configured via the configuration wizard, see [Run the accounting wizard](./01_RunAccountingWizard.md).
+- A valid fiscal year is created, see [Create a fiscal year](./04_ManageFiscalYear.md#create-a-fiscal-year).
+- A fixed booking batch is created, see [Create a fixed booking batch](#create-a-fixed-booking-batch).
+
+#### Procedure
+
+*Accounting > Select the fiscal year > Settings > Tab FIXED BOOKINGS*
+
+![Delete an fixed booking batch](../../Assets/Screenshots/RetailSuiteAccounting/Settings/FixedBookings/DeleteFixedBookingBatch.png "[Delete a fixed booking batch]")
+
+1. Click the *Fixed booking batch* drop-down list and select the fixed booking batch to be deleted.   
+
+  > [Warning] Be aware that the existing fixed booking batch will be deleted permanently. The bookings contained in it, however, will not be deleted from the system and will continue to be displayed in the *Bookings* tab.
+
+2. Click the [DELETE] button.  
+A window comes up asking for confirmation. The selected fixed booking batch is deleted.
+
+  > [Info] You may need to refresh the page with the F5 key to see the changes in the Fixed booking batch drop-down list.
+
+
+## Create a fixed booking
+
+You add fixed bookings to an existing or a newly created fixed booking batch.
+
+#### Prerequisites
+
+- The accounting is configured via the configuration wizard, see [Run the accounting wizard](./01_RunAccountingWizard.md).
+- A valid fiscal year is created, see [Create a fiscal year](./04_ManageFiscalYear.md#create-a-fiscal-year).
+- A fixed booking batch is created, see [Create a fixed booking batch](#create-a-fixed-booking-batch).
+
+#### Procedure
+
+*Accounting > Select the fiscal year > Settings > Tab FIXED BOOKINGS*
+
+![Select a fixed booking batch](../../Assets/Screenshots/RetailSuiteAccounting/Settings/FixedBookings/EditFixedBookingBatch.png "[Select a fixed booking batch]")
+
+1. Click the *Fixed booking batch* drop-down list and select the fixed booking batch where the fixed booking must be created.   
+The fixed booking window opens automatically. If it does not, click the [EDIT] button.
+
+  ![Create a fixed booking](../../Assets/Screenshots/RetailSuiteAccounting/Settings/FixedBookings/CreateFixedBooking.png "[Create a fixed booking]")
+
+2. Enter a value in the *Turnover* field.
+
+  > [Info] The booking numbers must be sequential and are added by the system automatically.
+
+3. Enter a contra account number in the *Contra account* field.
+
+  > [Info] When you start typing in an account number, a context menu is displayed. You can also select the appropriate option from the menu.
+
+4. Enter the date for the fixed booking in the *Date* field. Date format must be TTMMJJ or TT.MM.JJ.
+
+5. Enter an account number in the *Account* field.
+
```

Comment:
> [Info] After you have entered at least two numbers in the field, a list below the field displays all accounts that match the entry. You can also select the corresponding account in the list.